docs/examples/droplet_fusion/droplet_fusion.rst

```diff
@@ -0,0 +1,208 @@
+
+We will be fitting the force data during the fusion event with the following equation: :math:`f(t) = ae^{-t/\tau}+bt+c`
+
+The term $bt$ accounts for the movement of the trap, assuming a constant trap speed [1]_. The parameter of interest is $\tau$, the relaxation time schale of the fusion event::
+
+    from scipy.optimize import curve_fit
+
+    def relaxation_model(t, tau, a, b, c):
+        return a * np.exp(-t / tau) + b * t + c
+
+Fit the data and plot the result::
+
+    time = force_selection.seconds
+    force = force_selection.data
+
+    popt, pcov = curve_fit(relaxation_model, time, force, [0.1,force[0],0,0])
+    print(popt)
+    plt.figure()
+    plt.plot(time,force)
+    plt.plot(time,relaxation_model(time,*popt),label=f'$\\tau$ = {popt[0]:0.2f}s')
+    plt.legend()
+    plt.show()
+
+.. image:: fit.png
+
+The relaxation time obtained from the fit is 0.05 seconds. 
+
+Now we will proceed to determine the size of the droplets before the fusion event.
+
+Droplet size
+------------
+
+First load the scan and print the relevant metadata::
+
+    >>> for name, scan in f.scans.items():
+    >>>        print(f"num frames: {scan.num_frames}")
+    >>>        frame_duration = (scan.frame_timestamp_ranges()[0][1]-scan.frame_timestamp_ranges()[0][0])/1e9
+    >>>        print(f"frame duration: {frame_duration} s")
+
+    num frames: 27
+    frame duration: 0.4977664 s
+
+Plot a frame before the fusion event::
+
+    framenr = 2
+    plt.figure()
+    scan.plot(channel="green", frame=framenr)
+    plt.show()
+
+.. image:: frame2.png
+
+Plot a frame after the fusion event::
+
+    framenr = 6
+    plt.figure()
+    scan.plot(channel="green", frame=framenr)
+    plt.show()
+
+.. image:: frame6.png
+
+If the droplets are in focus, the size of the droplet can be estimated from the 2D scan. 
+The estimate will not be extremely precise, because the right droplet is moving as the scan is recorded, and the boarders of the scanned images are not sharp.
+The estimate in the size will therefore have an error of ~10%.
+
+The first step, is to use image segmentation to identify the two droplets in the image. 
+The threshold may need to be optimized for your data::
+
+    import math
+    from skimage.measure import label, regionprops
+
+    framenr = 2  # Choose a frame before the fusion event on which you want to identify and measure droplets
+
+    image = scan.get_image(channel='red')[framenr]
+    image = image / np.max(image)
+    threshold = 0.5
+    blobs = image > threshold
+    label_img = label(blobs)
+
+    plt.figure()
+    plt.subplot(2, 1, 1)
+    plt.title("original, normalized image")
+    plt.imshow(image)
+    plt.subplot(2, 1, 2)
+    plt.title(f"Identified objects")
+    plt.imshow(label_img)
+    plt.tight_layout()
+    plt.show()
+
+.. image:: image_segmentation.png
+
+Estimate the size of the droplet by measuring the length of the longest axis and the shortest axis and taking the average::
+
+    regions = regionprops(label_img)
```

Comment:
Since you can't be sure that the threshold works well for them, consider explicitly asserting that there are only two. Something like:
```python
if (blobs := len(np.unique(label_img))) != 3:
    raise RuntimeError(f"Expected 2 blobs, found {blobs - 1} instead! Maybe adjust the threshold?")
```